node/service/src/parachains_db/mod.rs

```diff
@@ -107,6 +113,33 @@ pub fn open_creating(root: PathBuf, cache_sizes: CacheSizes) -> io::Result<Arc<d
 	std::fs::create_dir_all(&path_str)?;
 	upgrade::try_upgrade_db(&path)?;
 	let db = Database::open(&db_config, &path_str)?;
+	let db =
+		polkadot_node_subsystem_util::database::kvdb_impl::DbAdapter::new(db, columns::ORDERED_COL);
+
+	Ok(Arc::new(db))
+}
+
+/// Open a parity db database.
+#[cfg(feature = "full-node")]
+pub fn open_creating(root: PathBuf, _cache_sizes: CacheSizes) -> io::Result<Arc<dyn Database>> {
+	let path = root.join("parachains");
```

Comment:
I don't remember well, but I had rfc from @arkpar on changing this (iirc just the fact that parachains are nested in rocksdb directory):
- chains/polkadot/paritydb/full
- chains/polkadot/paritydb/parachains
the idea iirc was to avoid nested directories (db containing parachains and db file in the same repo).
when rocksdb is currently:
- chains/polkadot/db/full
- chains/polkadot/db/full/parachains/db

So I think I can switch to:
- chains/polkadot/paritydb/full
- chains/polkadot/paritydb/parachains/db
but the added 'db' seems a bit redundant with 'paritydb', but in case over content would need to be under paritydb/parachains, I can do it?

Reply:
The long term plan is to get rid of rocksdb. Might as well switch to a better structure.

Reply:
Fair enough.